### DISCUSSION TITLE
Inclusion Beewi DD01A 18W35

### DISCUSSION THREAD

**Discussion by ghost** [theengs/decoder#213] [New devices/things integration]
Comment participé pour ajouté de nouveau périphériques ?

Hello

Comment ajouter ce module et le rendre compatible avec Theengs 

Beewi DD01A 18W35
BSDOO-EUR rev: AW11
3V CR2032

https://forum.hacf.fr/uploads/default/original/3X/8/9/89d7020ae78f7eb702e74bf36b0ca87312c4a7fe.jpeg
https://forum.hacf.fr/uploads/default/original/3X/8/9/89d7020ae78f7eb702e74bf36b0ca87312c4a7fe.jpeg

Cela permettrai de recyclé certain module 

Pour info le json vu par Mqtt Explorer.

Capteur Fermer:
```
{
  "manufacturerdata": "0d00080c000664",
  "name": "023600911459110836",
  "id": "30:45:11:E5:C7:61",
  "rssi": -42
  "rssi": -61
}
```

Capteur Ouvert

```
{
  "manufacturerdata": "0d00080c010664",
  "name": "023600911459110836",
  "id": "30:45:11:E5:C7:61",
  "rssi": -55
  "rssi": -58
}
```

seul la ligne **manufacturerdata** change : Ouvert/Fermer

Merci

**Comment by 1technophile** [comment]
Hello,

Merci, est ce que tu as un valeur de batterie egalement dans son application ou avec blea?
Si c'est le cas ce serait interessant de l'avoir et egalement de partager le manufacturerdata, la valeur dans l'app ou blea avec une batterie differente. Cela revient a completer les ? dans le tableau ci dessous

|State|Battery % or voltage|Manufacturerdata|
|-|-|-|
|close|?|0d00080c000664|
|open|?|0d00080c010664|
|close with another battery|?|?|
|open with another battery|?|?|

**Comment by ghost** [comment]
Re,

Alors l'appication officielle est celle ci
https://play.google.com/store/apps/details?id=com.otiohome&hl=fr

J'ai bien une remonté de la batterie dans l'application de otio ( petite image screem 2 )
![Screenshot_2022-10-12-23-00-06-525_com otiohome](https://user-images.githubusercontent.com/103725660/195452370-6435229c-b5fe-41d5-a664-11fb6034195b.jpg)
![Screenshot_2022-10-12-22-59-51-821_com otiohome](https://user-images.githubusercontent.com/103725660/195452531-27399da4-b17a-440b-aa9c-5ffd39b6d2b7.jpg)

![Screenshot_2022-10-12-23-00-13-036_com otiohome](https://user-images.githubusercontent.com/103725660/195452615-ceec9e14-63ab-46a2-b9c5-ba38ed6915fb.jpg)


Dans MQTT.Explorer , je n'ai aucune info de batterie.

![mqtt](https://user-images.githubusercontent.com/103725660/195453067-7f9cb157-c27b-4c5a-b4d5-490b08ff476f.PNG)


Sinon j'ai trouvé cela de jeedom (BLEA)
https://forum.jeedom.com/viewtopic.php?t=42441

Un Github 
https://github.com/jeedom/plugin-blea/pull/50/commits/375c4d8f4bd314cc017efc42b86f1e5670d5e016#diff-cdf9d028d5b01d7ecd117bf699461d1924104df2106b1dc00133693bfe31d469

https://github.com/jeedom/plugin-blea/pull/50/commits/f8ae9833e388b83346bd41a3c17a0651ec261cff

https://github.com/jeedom/plugin-blea/pull/50/commits/38a9efae55430cf7ae23ce0e8be66605213b9b7d

https://github.com/jeedom/plugin-blea/pull/50/commits/3a9010ecd79374d08a43dfbd5685b2efb5d32ba0

https://github.com/jeedom/plugin-blea/pull/50/commits/1a48204681e884f590e6a1b28c2a3c928b056aee

**Reply by DigiH** [reply]
https://github.com/theengs/decoder/pull/214

pour l'inclusion avec statut d'ouverture et pile.

**Reply by 1technophile** [reply]
@Sonic62 pourrais tu faire quelques tests avec des piles de différents voltages?
Cela permettrait de vérifier le décodage.

**Comment by ghost** [comment]
Hello

La seule pile que j'ai sous la main est de 2.80v cr2032 mesuré au voltmètre.

![6](https://user-images.githubusercontent.com/103725660/198372198-99c898da-efe4-472e-9f8b-b6eb382e82fa.PNG)
![7](https://user-images.githubusercontent.com/103725660/198372200-2fd7096d-7597-46fd-8006-bd7b01e348df.PNG)
![88](https://user-images.githubusercontent.com/103725660/198372205-fefbdecf-0b59-4cc7-a780-32a4b474c3c2.PNG)
![Capturevbxb](https://user-images.githubusercontent.com/103725660/198372206-ffdaa3a2-253a-4387-9fdc-30f04d84887a.PNG)

**Reply by 1technophile** [reply]
Merci, 2.8v correspond au 11 et 14% ?

**Comment by ghost** [comment]
Re, 

toutes les captures corresponde avec la même pile de 2,8v

Depuis hier j'ai laisse la pile de 2,8v 
![Capture](https://user-images.githubusercontent.com/103725660/198589595-01e0edb5-e880-498e-a266-f5c32a2a05f7.PNG)
